gif/Form1.cs

```diff
@@ -179,5 +191,80 @@ private void lstInfectedFiles_DoubleClick(object sender, EventArgs e)
                 System.Diagnostics.Process.Start("explorer.exe", args);
             }
         }
+
+        private void bwrLoadAddonList_DoWork(object sender, System.ComponentModel.DoWorkEventArgs e)
+        {
+            updateAddonList();
+        }
+
+        private void btnUpdateIDs_Click(object sender, EventArgs e)
```

Comment:
Rename the text for the update button. I was initially confused because I was expecting it to let me manually update/add IDs to the list, but found out it retrieves using the HTTP request to retrieve them from the file. This could be confusing to users if they don't already know.

Reply:
I'll go and update that as well